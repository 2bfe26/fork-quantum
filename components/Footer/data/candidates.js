export default {
  id: 1,
  title: 'Candidatos',
  cols: 8,
  theme: {
    primaryColor: '#005da4',
    secondaryColor: '#666666',
    titleFontSize: '14px',
    titleFontWeight: 'bold',
    linkFontSize: '12px',
  },
  linkGroups: [{
    id: 1,
    sideSeparator: true,
    cols: 4,
    links: [{
      id: 1,
      title: true,
      text: 'Buscar vagas',
      href: null,
      target: null,
      img: null,
    }, {
      id: 2,
      title: false,
      text: 'Por palavra-chave',
      href: '/vagas/',
      target: '_self',
      img: null,
    }, {
      id: 3,
      title: false,
      text: 'Por área/cargo',
      href: 'https://www.catho.com.br/vagas/por-area/',
      target: '_self',
      img: null,
    }, {
      id: 4,
      title: false,
      text: 'Melhores e maiores empresas do Brasil',
      href: 'https://www3.catho.com.br/vag/empregos-maiores-empresas/',
      target: '_self',
      img: null,
    }],
  }, {
    id: 2,
    sideSeparator: true,
    cols: 4,
    links: [{
      id: 1,
      title: true,
      text: 'Produtos Gratuitos',
      href: null,
      target: null,
      img: null,
    }, {
      id: 2,
      title: false,
      text: 'Aviso de vagas e buscas salvas',
      href: '/aviso-vagas/listagem/',
      target: '_self',
      img: null,
    }, {
      id: 3,
      title: false,
      text: 'Buscar vagas',
      href: 'https://www.catho.com.br/vagas/',
      target: '_self',
      img: null,
    }, {
      id: 4,
      title: false,
      text: 'Guia de Profissões e Salários',
      href: '/profissoes/',
      target: '_self',
      img: null,
    }, {
      id: 5,
      title: false,
      text: 'Por dentro das empresas',
      href: '/por-dentro/',
      target: '_self',
      img: null,
    }],
  }, {
    id: 3,
    sideSeparator: false,
    cols: 4,
    links: [{
      id: 1,
      title: true,
      text: 'Apoio ao candidato',
      href: null,
      target: null,
      img: null,
    }, {
      id: 2,
      title: false,
      text: 'Elaboração de Currículo',
      href: '/https://parceiros.catho.com.br/gop.php?origem=cat&a=6515&p=12&url1=1&url=servicoscatho%2Fcarreira%2Felaboracao-de-curriculo%2F',
      target: '_self',
      img: null,
    }, {
      id: 3,
      title: false,
      text: 'Análise de Currículo',
      href: 'https://parceiros.catho.com.br/gop.php?origem=cat&a=6523&p=4&url1=1&url=servicoscatho%2Fcarreira%2Fanalise-de-curriculo%2F',
      target: '_self',
      img: null,
    }, {
      id: 4,
      title: false,
      text: 'Simulação de Entrevista',
      href: 'https://parceiros.catho.com.br/gop.php?origem=cat&a=6527&p=16&url1=1&url=servicoscatho%2Fcarreira%2Fsimulacao-de-entrevista%2F',
      target: '_self',
      img: null,
    }, {
      id: 5,
      title: false,
      text: 'Elaboração de Carta de Apresentação',
      href: 'https://parceiros.catho.com.br/gop.php?origem=cat&a=6519&p=13&url1=1&url=servicoscatho%2Fcarreira%2Fcarta-de-apresentacao%2F',
      target: '_self',
      img: null,
    }, {
      id: 6,
      title: false,
      text: 'Conheça Todas as Soluções',
      href: '/servicoscandidatos/',
      target: '_self',
      img: null,
    }],
  }, {
    id: 4,
    sideSeparator: true,
    cols: 4,
    links: [{
      id: 1,
      title: true,
      text: 'Carreira e sucesso',
      href: 'https://www.catho.com.br/carreira-sucesso/',
      target: '_self',
      img: null,
    }],
  }, {
    id: 5,
    sideSeparator: true,
    cols: 4,
    links: [{
      id: 1,
      title: true,
      text: 'Exclusivo para assinantes',
      href: null,
      target: null,
      img: null,
    }, {
      id: 2,
      title: false,
      text: 'Cursos Rápidos',
      href: '/cursos-rapidos/',
      target: '_self',
      img: null,
    }, {
      id: 3,
      title: false,
      text: 'Cadastro Catho',
      href: 'https://cadastro.catho.com.br/',
      target: '_self',
      img: null,
    }, {
      id: 4,
      title: false,
      text: 'Consultor Virtual',
      href: '/consvirtual/',
      target: '_self',
      img: null,
    }],
  }, {
    id: 6,
    sideSeparator: false,
    cols: 4,
    links: [{
      id: 1,
      title: true,
      text: 'Ajuda',
      href: null,
      target: null,
      img: null,
    }, {
      id: 2,
      title: false,
      text: 'Central de Relacionamento',
      href: 'https://www.catho.com.br/sac/',
      target: '_self',
      img: null,
    }, {
      id: 3,
      title: true,
      text: 'Promoção 7 dias grátis',
      href: null,
      target: null,
      img: null,
    }, {
      id: 5,
      title: false,
      text: 'Entenda a promoção',
      href: 'https://www.catho.com.br/help/usr/faq_promocao.phtml?home=1&width=650px&height=600px&iframe=true',
      target: '_self',
      img: null,
    }],
  }],
};
