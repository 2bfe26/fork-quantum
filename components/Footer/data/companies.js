export default {
  id: 1,
  title: 'Empresas',
  cols: 2,
  theme: {
    primaryColor: '#005C45',
    secondaryColor: '#666666',
    titleFontSize: '14px',
    titleFontWeight: 'bold',
    linkFontSize: '12px',
  },
  linkGroups: [{
    id: 1,
    sideSeparator: false,
    cols: 12,
    links: [{
      id: 1,
      title: false,
      text: 'Solução Atração dos Melhores',
      href: 'https://seguro.catho.com.br/inc/novaInclusao/FormPrincipal.php?tipoProd=Vag&c=13006',
      target: '_self',
      img: null,
    }, {
      id: 2,
      title: false,
      text: 'Solução Recrutamento Perfeito',
      href: 'https://www.catho.com.br/empresas/produtos/',
      target: '_self',
      img: null,
    }, {
      id: 3,
      title: false,
      text: 'Produtos para Empresa',
      href: 'https://www.catho.com.br/empresas/produtos/',
      target: '_self',
      img: null,
    }],
  }],
};
