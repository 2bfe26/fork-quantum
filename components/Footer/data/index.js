import Candidates from './candidates';
import Companies from './companies';
import Courses from './courses';
import Institutional from './institutional';
import Stamps from './stamps';

const Data = {
  theme: {
    primaryColor: '#005da4',
    secondaryColor: '#666666',
    borderColor: '#e3e3e3',
    backgroundColor: '#f5f5f5',
    fontFamily: 'Arial',
    fontSize: '12px',
    paddingTop: '15px',
    paddingBottom: '10px',
    StampsHolderPadding: '10px',
  },
  candidates: Candidates,
  companies: Companies,
  courses: Courses,
  institutional: Institutional,
  stamps: Stamps,
};

export default Data;
