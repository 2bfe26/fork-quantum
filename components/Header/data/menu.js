const common = [
  {
    id: 3,
    title: false,
    text: 'Aviso de vagas',
    href: '/aviso-vagas/listagem/',
    target: '_self',
    img: null,
  }, {
    id: 4,
    title: false,
    text: 'Envie seu Currículo por 7 dias grátis',
    href: 'https://seguro.catho.com.br/inclusao/curriculo.php?nova_inclusao=1&usr[plano_id]=1&cur[perfil_id]=1&cur[use_personalizado]=0&a=0&ppd=0&tipoinclusao=807',
    target: '_self',
    img: null,
  }, {
    id: 5,
    title: false,
    text: 'Guia de profissões',
    href: '/profissoes/',
    target: '_self',
    img: null,
  }, {
    id: 6,
    title: null,
    text: 'Por dentro das empresas',
    href: '/por-dentro/',
    target: '_self',
    img: null,
  }, {
    id: 7,
    title: null,
    text: 'Portal do currículo',
    href: 'https://www.catho.com.br/portaldocurriculo/',
    target: '_self',
    img: null,
  }, {
    id: 8,
    title: null,
    text: 'Vagas de estágio',
    href: 'https://www.catho.com.br/estagio/',
    target: '_self',
    img: null,
  }, {
    id: 9,
    title: 'Exclusivo para assinantes',
    text: null,
    href: null,
    target: null,
    img: null,
  }, {
    id: 10,
    title: null,
    text: 'Cursos rápidos',
    href: '/cursos-rapidos/',
    target: '_self',
    img: null,
  }, {
    id: 11,
    title: null,
    text: 'Cadastro Catho',
    href: 'https://cadastro.catho.com.br/',
    target: '_self',
    img: null,
  }, {
    id: 12,
    title: null,
    text: 'Consultor virtual',
    href: '/consvirtual/',
    target: '_self',
    img: null,
  }, {
    id: 13,
    title: null,
    text: 'Vantagens de assinar a Catho',
    href: '/cadastrar-curriculo/',
    target: '_self',
    img: null,
  }, {
    id: 14,
    title: 'Apoio ao candidato',
    text: null,
    href: null,
    target: null,
    img: null,
  }, {
    id: 15,
    title: null,
    text: 'Elaboração de currículo',
    href: 'https://parceiros.catho.com.br/gop.php?origem=cat&a=6503&p=12&url1=1&url=servicoscatho/carreira/elaboracao-de-curriculo/',
    target: '_self',
    img: null,
  }, {
    id: 16,
    title: null,
    text: 'Análise de currículo',
    href: 'https://parceiros.catho.com.br/gop.php?origem=cat&a=6507&p=4&url1=1&url=servicoscatho/carreira/analise-de-curriculo/',
    target: '_self',
    img: null,
  }, {
    id: 17,
    title: null,
    text: 'Simulação de entrevista',
    href: 'https://parceiros.catho.com.br/gop.php?origem=cat&a=6509&p=16&url1=1&url=servicoscatho/carreira/simulacao-de-entrevista/',
    target: '_self',
    img: null,
  }, {
    id: 18,
    title: null,
    text: 'Elaboração de carta de apresentação',
    href: 'https://parceiros.catho.com.br/gop.php?origem=cat&a=6505&p=13&url1=1&url=servicoscatho/carreira/carta-de-apresentacao/',
    target: '_self',
    img: null,
  }, {
    id: 19,
    title: null,
    text: 'Cursos',
    href: '/curso/',
    target: '_self',
    img: null,
  }, {
    id: 20,
    title: null,
    text: 'Todas as soluções',
    href: '/servicoscandidatos/',
    target: '_self',
    img: null,
  },
];

const desktop = {
  theme: {
    primaryColor: '#666666',
    secondaryColor: '#CCCCCC',
    tertiaryColor: '#333333',
    backgroundColor: '#FFFFFF',
    titleFontSize: '14px',
    titleFontWeight: 'bold',
    linkFontSize: '14px',
    marginLogo: '0px 20px 0px 0px',
    menuHover: {
      borderColor: '#005da4',
      borderSize: '2px',
    },
  },
  links: [{
    id: 1,
    title: false,
    text: 'Buscar vagas',
    href: '/vagas/',
    target: '_self',
    img: null,
  }, {
    id: 2,
    title: false,
    text: 'Guia de profissões',
    href: '/profissoes/',
    target: '_self',
    img: null,
  }, {
    id: 3,
    title: null,
    text: 'Por dentro das empresas',
    href: '/por-dentro/',
    target: '_self',
    img: null,
  }, {
    id: 4,
    title: false,
    text: 'Soluções para candidatos',
    href: null,
    target: null,
    img: null,
    collapsedLinkGroup: common,
  }],
};

const phone = {
  theme: {
    primaryColor: '#666666',
    secondaryColor: '#CCCCCC',
    tertiaryColor: '#333333',
    titleFontSize: '14px',
    titleFontWeight: 'bold',
    linkFontSize: '12px',
  },
  links: [{
    id: 1,
    title: false,
    text: 'Buscar vagas',
    href: '/vagas/',
    target: '_self',
    img: null,
    icon: 'search',
  }, {
    id: 2,
    title: null,
    text: 'Por dentro das empresas',
    href: '/por-dentro/',
    target: '_self',
    img: null,
    icon: 'enterprise',
  }, {
    id: 3,
    title: false,
    text: 'Aviso de vagas',
    href: '/aviso-vagas/listagem/',
    target: '_self',
    img: null,
    icon: 'alert',
  },
  ...common.slice(1)],
};

export default {
  desktop,
  phone,
};
