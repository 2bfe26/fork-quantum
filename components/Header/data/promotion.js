export default {
  theme: {
    primaryColor: '#da1d5d',
    secondaryColor: '#b6154e',
    textColor: '#fff',
  },
};
