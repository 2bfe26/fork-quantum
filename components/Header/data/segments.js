export default {
  id: 1,
  title: null,
  cols: 12,
  theme: {
    primaryColor: '#555',
    secondaryColor: '#CCCCCC',
    tertiaryColor: '#333333',
    titleFontSize: '14px',
    titleFontWeight: 'bold',
    linkFontSize: '12px',
  },
  links: [{
    id: 1,
    title: false,
    text: 'candidatos',
    href: '/',
    target: '_self',
    img: null,
  }, {
    id: 2,
    title: false,
    text: 'empresas',
    href: 'https://www.catho.com.br/empresa/',
    target: '_self',
    img: null,
  }, {
    id: 3,
    title: false,
    text: 'cursos',
    href: 'https://www.catho.com.br/educacao/',
    target: '_self',
    img: null,
  }, {
    id: 4,
    title: false,
    text: null,
    href: 'https://www.catho.com.br/help/',
    target: '_self',
    img: {
      src: 'header-info.svg',
      width: '16px',
      height: '16px',
      alt: '',
    },
  }],
};
