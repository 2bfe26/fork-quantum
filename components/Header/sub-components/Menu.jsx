import React from 'react';
import PropTypes from 'prop-types';
import MenuDesktop from './MenuDesktop';
import MenuMobile from './MenuMobile';

const Menu = ({
  data, handleMenuOpen, handleLoginOpen, menuOpened, loginOpened,
}) => (
  <section>
    <MenuMobile data={data} menuOpened={menuOpened} />
    <MenuDesktop
      data={data}
      handleLoginOpen={handleLoginOpen}
      handleMenuOpen={handleMenuOpen}
      menuOpened={menuOpened}
      loginOpened={loginOpened}
    />
  </section>
);

Menu.propTypes = {
  data: PropTypes.instanceOf(Object),
  handleMenuOpen: PropTypes.func.isRequired,
  handleLoginOpen: PropTypes.func.isRequired,
  menuOpened: PropTypes.string,
  loginOpened: PropTypes.bool,
};

Menu.defaultProps = {
  data: null,
  menuOpened: null,
  loginOpened: false,
};

export default Menu;
