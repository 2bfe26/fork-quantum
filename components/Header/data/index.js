import Segments from './segments';
import Promotion from './promotion';
import Menu from './menu';

const Data = {
  theme: {
    primaryColor: '#005da4',
    secondaryColor: '#666666',
    borderColor: '#e3e3e3',
    backgroundColor: '#f5f5f5',
    fontFamily: 'Arial',
    fontSize: '12px',
    paddingTop: '10px',
    paddingBottom: '10px',
    StampsHolderPadding: '10px',
  },
  segments: Segments,
  promotion: Promotion,
  menu: Menu,
};

export default Data;
