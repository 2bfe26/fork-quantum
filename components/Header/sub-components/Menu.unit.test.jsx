import React from 'react';
import renderer from 'react-test-renderer';
import Menu from './Menu';
import data from '../data';

describe('Menu component', () => {
  it('Should match the snapshot', () => {
    const tree = renderer.create(<Menu
      data={data}
      handleMenuOpen={() => {}}
      handleLoginOpen={() => {}}
      menuOpened={null}
      loginOpened={false}
    />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
