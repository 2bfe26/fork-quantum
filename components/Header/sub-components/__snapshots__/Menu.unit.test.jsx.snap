// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu component Should match the snapshot 1`] = `
<section>
  <div
    className="MenuMobile__MenuMobileContainer-dztz2f-0 cDxolp Container-s163g650-0 jvPZlp"
  >
    <div
      className="Row-k9527m-0 kvCkwg"
    >
      <div
        className="Col-s1jhptn4-0 fgbnAC"
      >
        <div
          className="MenuMobile__FeaturedListItem-dztz2f-4 gXyags"
        >
          <a
            className="MenuMobile__FeaturedLink-dztz2f-5 fHwgtd"
            href="/vagas/"
            icon="search"
            title="Buscar vagas"
          >
            Buscar vagas
          </a>
        </div>
      </div>
      <div
        className="Col-s1jhptn4-0 fgbnAC"
      >
        <div
          className="MenuMobile__FeaturedListItem-dztz2f-4 gXyags"
        >
          <a
            className="MenuMobile__FeaturedLink-dztz2f-5 kZNqVj"
            href="/por-dentro/"
            icon="enterprise"
            title="Por dentro das empresas"
          >
            Por dentro das empresas
          </a>
        </div>
      </div>
      <div
        className="Col-s1jhptn4-0 fgbnAC"
      >
        <div
          className="MenuMobile__FeaturedListItem-dztz2f-4 gXyags"
        >
          <a
            className="MenuMobile__FeaturedLink-dztz2f-5 cTNrFd"
            href="/aviso-vagas/listagem/"
            icon="alert"
            title="Aviso de vagas"
          >
            Aviso de vagas
          </a>
        </div>
      </div>
      <ul
        className="MenuMobile__LinkList-dztz2f-1 cgZdXB"
      >
        <li
          className="MenuMobile__LinkListItem-dztz2f-2 djBawj"
        >
          <a
            className="MenuMobile__Link-dztz2f-3 ceVGrD"
            href="https://seguro.catho.com.br/inclusao/curriculo.php?nova_inclusao=1&usr[plano_id]=1&cur[perfil_id]=1&cur[use_personalizado]=0&a=0&ppd=0&tipoinclusao=807"
            target="_self"
            title="Envie seu Currículo por 7 dias grátis"
          >
            Envie seu Currículo por 7 dias grátis
          </a>
        </li>
        <li
          className="MenuMobile__LinkListItem-dztz2f-2 djBawj"
        >
          <a
            className="MenuMobile__Link-dztz2f-3 ceVGrD"
            href="/profissoes/"
            target="_self"
            title="Guia de profissões"
          >
            Guia de profissões
          </a>
        </li>
        <li
          className="MenuMobile__LinkListItem-dztz2f-2 djBawj"
        >
          <a
            className="MenuMobile__Link-dztz2f-3 ceVGrD"
            href="/por-dentro/"
            target="_self"
            title="Por dentro das empresas"
          >
            Por dentro das empresas
          </a>
        </li>
        <li
          className="MenuMobile__LinkListItem-dztz2f-2 djBawj"
        >
          <a
            className="MenuMobile__Link-dztz2f-3 ceVGrD"
            href="https://www.catho.com.br/portaldocurriculo/"
            target="_self"
            title="Portal do currículo"
          >
            Portal do currículo
          </a>
        </li>
        <li
          className="MenuMobile__LinkListItem-dztz2f-2 djBawj"
        >
          <a
            className="MenuMobile__Link-dztz2f-3 ceVGrD"
            href="https://www.catho.com.br/estagio/"
            target="_self"
            title="Vagas de estágio"
          >
            Vagas de estágio
          </a>
        </li>
        <li
          className="MenuMobile__LinkListItem-dztz2f-2 ilBnDi"
        >
          Exclusivo para assinantes
        </li>
        <li
          className="MenuMobile__LinkListItem-dztz2f-2 djBawj"
        >
          <a
            className="MenuMobile__Link-dztz2f-3 ceVGrD"
            href="/cursos-rapidos/"
            target="_self"
            title="Cursos rápidos"
          >
            Cursos rápidos
          </a>
        </li>
        <li
          className="MenuMobile__LinkListItem-dztz2f-2 djBawj"
        >
          <a
            className="MenuMobile__Link-dztz2f-3 ceVGrD"
            href="https://cadastro.catho.com.br/"
            target="_self"
            title="Cadastro Catho"
          >
            Cadastro Catho
          </a>
        </li>
        <li
          className="MenuMobile__LinkListItem-dztz2f-2 djBawj"
        >
          <a
            className="MenuMobile__Link-dztz2f-3 ceVGrD"
            href="/consvirtual/"
            target="_self"
            title="Consultor virtual"
          >
            Consultor virtual
          </a>
        </li>
        <li
          className="MenuMobile__LinkListItem-dztz2f-2 djBawj"
        >
          <a
            className="MenuMobile__Link-dztz2f-3 ceVGrD"
            href="/cadastrar-curriculo/"
            target="_self"
            title="Vantagens de assinar a Catho"
          >
            Vantagens de assinar a Catho
          </a>
        </li>
        <li
          className="MenuMobile__LinkListItem-dztz2f-2 ilBnDi"
        >
          Apoio ao candidato
        </li>
        <li
          className="MenuMobile__LinkListItem-dztz2f-2 djBawj"
        >
          <a
            className="MenuMobile__Link-dztz2f-3 ceVGrD"
            href="https://parceiros.catho.com.br/gop.php?origem=cat&a=6503&p=12&url1=1&url=servicoscatho/carreira/elaboracao-de-curriculo/"
            target="_self"
            title="Elaboração de currículo"
          >
            Elaboração de currículo
          </a>
        </li>
        <li
          className="MenuMobile__LinkListItem-dztz2f-2 djBawj"
        >
          <a
            className="MenuMobile__Link-dztz2f-3 ceVGrD"
            href="https://parceiros.catho.com.br/gop.php?origem=cat&a=6507&p=4&url1=1&url=servicoscatho/carreira/analise-de-curriculo/"
            target="_self"
            title="Análise de currículo"
          >
            Análise de currículo
          </a>
        </li>
        <li
          className="MenuMobile__LinkListItem-dztz2f-2 djBawj"
        >
          <a
            className="MenuMobile__Link-dztz2f-3 ceVGrD"
            href="https://parceiros.catho.com.br/gop.php?origem=cat&a=6509&p=16&url1=1&url=servicoscatho/carreira/simulacao-de-entrevista/"
            target="_self"
            title="Simulação de entrevista"
          >
            Simulação de entrevista
          </a>
        </li>
        <li
          className="MenuMobile__LinkListItem-dztz2f-2 djBawj"
        >
          <a
            className="MenuMobile__Link-dztz2f-3 ceVGrD"
            href="https://parceiros.catho.com.br/gop.php?origem=cat&a=6505&p=13&url1=1&url=servicoscatho/carreira/carta-de-apresentacao/"
            target="_self"
            title="Elaboração de carta de apresentação"
          >
            Elaboração de carta de apresentação
          </a>
        </li>
        <li
          className="MenuMobile__LinkListItem-dztz2f-2 djBawj"
        >
          <a
            className="MenuMobile__Link-dztz2f-3 ceVGrD"
            href="/curso/"
            target="_self"
            title="Cursos"
          >
            Cursos
          </a>
        </li>
        <li
          className="MenuMobile__LinkListItem-dztz2f-2 djBawj"
        >
          <a
            className="MenuMobile__Link-dztz2f-3 ceVGrD"
            href="/servicoscandidatos/"
            target="_self"
            title="Todas as soluções"
          >
            Todas as soluções
          </a>
        </li>
      </ul>
    </div>
  </div>
  <div
    className="MenuDesktop__MenuDesktopContainer-s1xnd3dn-0 dnPXqP Container-s163g650-0 jvPZlp"
  >
    <div
      className="Container-s163g650-0 VziTO"
    >
      <div
        className="Row-k9527m-0 gKphLh"
      >
        <div
          className="MenuDesktop__StyledCol-s1xnd3dn-3 dnOJIb Col-s1jhptn4-0 gaTaBJ"
        >
          <div
            className="MenuDesktop__MenuNavigation-s1xnd3dn-4 ljJuGV"
          >
            <div
              className="Hide-s1pywwyr-0 cnRMmH"
            >
              <div
                className="MenuButton__Hamburger-qwwwlp-0 hhEvgD"
                onClick={[Function]}
              >
                <span
                  className="MenuButton__TopBread-qwwwlp-1 ggFiGA"
                />
                <span
                  className="MenuButton__Meat-qwwwlp-2 fSveOY"
                />
                <span
                  className="MenuButton__BottomBread-qwwwlp-3 cxgZPF"
                />
              </div>
            </div>
            <a
              href="https://www.catho.com.br/"
            >
              <img
                alt="A gente trabalha pelo seu trabalho"
                className="MenuDesktop__LogoCatho-s1xnd3dn-1 cDGNZP"
                src="https://static.catho.com.br/svg/site/logoCathoB2c.svg"
              />
            </a>
            <div
              className="MenuDesktop__StyledHide-s1xnd3dn-2 hYYQv Hide-s1pywwyr-0 jOgZNl"
            >
              <ul
                className="MenuDesktop__LinkList-s1xnd3dn-5 ldVNbb"
              >
                <li
                  className="MenuDesktop__LinkListItem-s1xnd3dn-6 cgbGWR"
                >
                  <a
                    className="MenuDesktop__Link-s1xnd3dn-7 iqrhBS"
                    href="/vagas/"
                  >
                    Buscar vagas
                  </a>
                </li>
                <li
                  className="MenuDesktop__LinkListItem-s1xnd3dn-6 cgbGWR"
                >
                  <a
                    className="MenuDesktop__Link-s1xnd3dn-7 iqrhBS"
                    href="/profissoes/"
                  >
                    Guia de profissões
                  </a>
                </li>
                <li
                  className="MenuDesktop__LinkListItem-s1xnd3dn-6 cgbGWR"
                >
                  <a
                    className="MenuDesktop__Link-s1xnd3dn-7 iqrhBS"
                    href="/por-dentro/"
                  >
                    Por dentro das empresas
                  </a>
                </li>
                <li
                  className="MenuDesktop__LinkListItem-s1xnd3dn-6 cgbGWR"
                >
                  Soluções para candidatos
                  <ul
                    className="MenuDesktop__PopOverLinkList-s1xnd3dn-8 kgmWSr"
                  >
                    <li
                      className="MenuDesktop__PopOverLinkListItem-s1xnd3dn-9 bjZTlf"
                    >
                      <a
                        className="MenuDesktop__PopOverLink-s1xnd3dn-11 DBdHB"
                        href="/aviso-vagas/listagem/"
                      >
                        Aviso de vagas
                      </a>
                    </li>
                    <li
                      className="MenuDesktop__PopOverLinkListItem-s1xnd3dn-9 bjZTlf"
                    >
                      <a
                        className="MenuDesktop__PopOverLink-s1xnd3dn-11 DBdHB"
                        href="https://seguro.catho.com.br/inclusao/curriculo.php?nova_inclusao=1&usr[plano_id]=1&cur[perfil_id]=1&cur[use_personalizado]=0&a=0&ppd=0&tipoinclusao=807"
                      >
                        Envie seu Currículo por 7 dias grátis
                      </a>
                    </li>
                    <li
                      className="MenuDesktop__PopOverLinkListItem-s1xnd3dn-9 bjZTlf"
                    >
                      <a
                        className="MenuDesktop__PopOverLink-s1xnd3dn-11 DBdHB"
                        href="/profissoes/"
                      >
                        Guia de profissões
                      </a>
                    </li>
                    <li
                      className="MenuDesktop__PopOverLinkListItem-s1xnd3dn-9 bjZTlf"
                    >
                      <a
                        className="MenuDesktop__PopOverLink-s1xnd3dn-11 DBdHB"
                        href="/por-dentro/"
                      >
                        Por dentro das empresas
                      </a>
                    </li>
                    <li
                      className="MenuDesktop__PopOverLinkListItem-s1xnd3dn-9 bjZTlf"
                    >
                      <a
                        className="MenuDesktop__PopOverLink-s1xnd3dn-11 DBdHB"
                        href="https://www.catho.com.br/portaldocurriculo/"
                      >
                        Portal do currículo
                      </a>
                    </li>
                    <li
                      className="MenuDesktop__PopOverLinkListItem-s1xnd3dn-9 bjZTlf"
                    >
                      <a
                        className="MenuDesktop__PopOverLink-s1xnd3dn-11 DBdHB"
                        href="https://www.catho.com.br/estagio/"
                      >
                        Vagas de estágio
                      </a>
                    </li>
                    <li
                      className="MenuDesktop__PopOverLinkListItem-s1xnd3dn-9 bjZTlf"
                    >
                      <span
                        className="MenuDesktop__PopOverTitle-s1xnd3dn-10 gZwjJ"
                      >
                        Exclusivo para assinantes
                      </span>
                    </li>
                    <li
                      className="MenuDesktop__PopOverLinkListItem-s1xnd3dn-9 bjZTlf"
                    >
                      <a
                        className="MenuDesktop__PopOverLink-s1xnd3dn-11 DBdHB"
                        href="/cursos-rapidos/"
                      >
                        Cursos rápidos
                      </a>
                    </li>
                    <li
                      className="MenuDesktop__PopOverLinkListItem-s1xnd3dn-9 bjZTlf"
                    >
                      <a
                        className="MenuDesktop__PopOverLink-s1xnd3dn-11 DBdHB"
                        href="https://cadastro.catho.com.br/"
                      >
                        Cadastro Catho
                      </a>
                    </li>
                    <li
                      className="MenuDesktop__PopOverLinkListItem-s1xnd3dn-9 bjZTlf"
                    >
                      <a
                        className="MenuDesktop__PopOverLink-s1xnd3dn-11 DBdHB"
                        href="/consvirtual/"
                      >
                        Consultor virtual
                      </a>
                    </li>
                    <li
                      className="MenuDesktop__PopOverLinkListItem-s1xnd3dn-9 bjZTlf"
                    >
                      <a
                        className="MenuDesktop__PopOverLink-s1xnd3dn-11 DBdHB"
                        href="/cadastrar-curriculo/"
                      >
                        Vantagens de assinar a Catho
                      </a>
                    </li>
                    <li
                      className="MenuDesktop__PopOverLinkListItem-s1xnd3dn-9 bjZTlf"
                    >
                      <span
                        className="MenuDesktop__PopOverTitle-s1xnd3dn-10 gZwjJ"
                      >
                        Apoio ao candidato
                      </span>
                    </li>
                    <li
                      className="MenuDesktop__PopOverLinkListItem-s1xnd3dn-9 bjZTlf"
                    >
                      <a
                        className="MenuDesktop__PopOverLink-s1xnd3dn-11 DBdHB"
                        href="https://parceiros.catho.com.br/gop.php?origem=cat&a=6503&p=12&url1=1&url=servicoscatho/carreira/elaboracao-de-curriculo/"
                      >
                        Elaboração de currículo
                      </a>
                    </li>
                    <li
                      className="MenuDesktop__PopOverLinkListItem-s1xnd3dn-9 bjZTlf"
                    >
                      <a
                        className="MenuDesktop__PopOverLink-s1xnd3dn-11 DBdHB"
                        href="https://parceiros.catho.com.br/gop.php?origem=cat&a=6507&p=4&url1=1&url=servicoscatho/carreira/analise-de-curriculo/"
                      >
                        Análise de currículo
                      </a>
                    </li>
                    <li
                      className="MenuDesktop__PopOverLinkListItem-s1xnd3dn-9 bjZTlf"
                    >
                      <a
                        className="MenuDesktop__PopOverLink-s1xnd3dn-11 DBdHB"
                        href="https://parceiros.catho.com.br/gop.php?origem=cat&a=6509&p=16&url1=1&url=servicoscatho/carreira/simulacao-de-entrevista/"
                      >
                        Simulação de entrevista
                      </a>
                    </li>
                    <li
                      className="MenuDesktop__PopOverLinkListItem-s1xnd3dn-9 bjZTlf"
                    >
                      <a
                        className="MenuDesktop__PopOverLink-s1xnd3dn-11 DBdHB"
                        href="https://parceiros.catho.com.br/gop.php?origem=cat&a=6505&p=13&url1=1&url=servicoscatho/carreira/carta-de-apresentacao/"
                      >
                        Elaboração de carta de apresentação
                      </a>
                    </li>
                    <li
                      className="MenuDesktop__PopOverLinkListItem-s1xnd3dn-9 bjZTlf"
                    >
                      <a
                        className="MenuDesktop__PopOverLink-s1xnd3dn-11 DBdHB"
                        href="/curso/"
                      >
                        Cursos
                      </a>
                    </li>
                    <li
                      className="MenuDesktop__PopOverLinkListItem-s1xnd3dn-9 bjZTlf"
                    >
                      <a
                        className="MenuDesktop__PopOverLink-s1xnd3dn-11 DBdHB"
                        href="/servicoscandidatos/"
                      >
                        Todas as soluções
                      </a>
                    </li>
                  </ul>
                </li>
              </ul>
            </div>
          </div>
          <ul
            className="LoginBox__LinkList-v6shrb-0 dFtQbH"
          >
            <li
              className="LoginBox__ListItem-v6shrb-1 ccvYWg"
            >
              <a
                className="LoginBox__Link-v6shrb-2 jFulmx"
                href="https://seguro.catho.com.br/inclusao/curriculo.php?nova_inclusao=1&usr%5Bplano_id%5D=1&cur%5Bperfil_id%5D=1&cur%5Buse_personalizado%5D=0&a=0&ppd=0&tipoinclusao=903"
              >
                Assine
              </a>
            </li>
            <li
              className="LoginBox__ListItem-v6shrb-1 ccvYWg"
            >
              <button
                className="LoginBox__Button-v6shrb-3 hltamn"
                onClick={[Function]}
              >
                Login
              </button>
              <div
                className="LoginBox__LoginFrame-v6shrb-4 hwNUsp"
              >
                <iframe
                  frameBorder="0"
                  scrolling="no"
                  src="https://safe.catho.com.br/login/tipo/tooltip/"
                  title="LoginBox"
                />
              </div>
            </li>
          </ul>
        </div>
      </div>
    </div>
  </div>
</section>
`;
